python/tests/test_options.py

```diff
@@ -27,98 +31,165 @@ def test_accepts_browser_version_platform_name(self):
         assert sauce.platform_name == 'macOS 10.13'
 
     def test_accepts_w3c_values(self):
-        options = {
-            "browserName": "firefox",
-            "sauce:options": {
-                "browserName": "firefox",
-                "browserVersion": "73.0",
-                "platformName": "Windows 8.1"
-            }
-        }
+        sauce = SauceOptions(acceptInsecureCerts=True, pageLoadStrategy='eager')
+
+        assert sauce.accept_insecure_certs == True
+        assert sauce.page_load_strategy == 'eager'
+
+    def test_accepts_sauce_values_with_dict(self):
+        options = {'maxDuration': 1,
+                   'commandTimeout': 2,
+                   'idleTimeout': 3,
+                   'name': 'foo',
+                   'build': 'bar',
+                   'tags': ['foo', 'bar'],
+                   'parentTunnel': 'bar',
+                   'tunnelIdentifier': 'foobar',
+                   'screenResolution': '10x10',
+                   'timeZone': 'Foo',
+                   'extendedDebugging': True,
+                   'capturePerformance': True,
+                   'recordVideo': False,
+                   'videoUploadOnPass': False,
+                   'recordScreenshots': False,
+                   'recordLogs': False,
+                   'username': 'foo',
+                   'accessKey': '1234',
+                   'passed': True}
+
+        sauce = SauceOptions(**options)
+
+        assert sauce.max_duration == 1
+        assert sauce.command_timeout == 2
+        assert sauce.idle_timeout == 3
+        assert sauce.name == 'foo'
+        assert sauce.build == 'bar'
+        assert sauce.tags == ['foo', 'bar']
+        assert sauce.parent_tunnel == 'bar'
+        assert sauce.tunnel_identifier == 'foobar'
+        assert sauce.screen_resolution == '10x10'
+        assert sauce.time_zone == 'Foo'
+        assert sauce.extended_debugging == True
+        assert sauce.capture_performance == True
+        assert sauce.record_video == False
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4711738)